code/modules/cm_phone/phone.dm

```diff
@@ -72,13 +75,15 @@ GLOBAL_LIST_EMPTY_TYPED(transmitters, /obj/structure/transmitter)
 	|| !T.enabled\
 )
 
-/proc/get_transmitters()
+/obj/structure/transmitter/proc/get_transmitters()
 	var/list/phone_list = list()
 
 	for(var/t in GLOB.transmitters)
 		var/obj/structure/transmitter/T = t
 		if(TRANSMITTER_UNAVAILABLE(T) || !T.callable) // Phone not available
 			continue
+		if(!(T.network_receive in networks_transmit))
+			continue
```

Comment:
why does this use t and T both as different variables dear god the unreadability

Reply:
Glad I can say it wasn't me at least!